python/es_multisink.py

```diff
@@ -23,17 +23,73 @@ def __init__(self, io_sig=1*[gr.sizeof_gr_complex], nthreads=8, samplehist=1000,
         # create sinks
         self.sinks = [];
         for i in range(0, nsinks):
+            if distribute:
+                gid = '{}_{}'.format(tg, i)
+            else:
+                gid = tg
```

Comment:
This line provides the same group ID to all sinks when the distributor is not connected.